### PR TITLE
Add git and linux basics

### PR DESCRIPTION
**Pull request by narra96**

### CODE REVIEW

File: git-basics-narra/git-narra.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+git clone : clone the project
+
```
Comment (manoj4971) [inline git-basics-narra/git-narra.txt]: remove space here

**Review by manoj4971 — COMMENTED** [review]
Provided comments , please check

File: git-basics-narra/git-narra.txt (hunk 1 above)
Comment (narra96) [reply git-basics-narra/git-narra.txt]: Done

**Review by narra96 — COMMENTED** [review]
Please review again.

**Pull request merged**